recipes/sdl_ttf/all/conanfile.py

```diff
@@ -82,22 +98,28 @@ def build(self):
 
     def package(self):
         copy(self, "COPYING.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         cmake = CMake(self)
         cmake.install()
         rmdir(self, os.path.join(self.package_folder, "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         rmdir(self, os.path.join(self.package_folder, "SDL2_ttf.framework"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
 
     def package_info(self):
+        suffix = "-static" if Version(self.version) >= "2.20.0" and not self.options.shared else ""
```

Comment:
Only for the well-beloved MSVC :)
```suggestion
        suffix = "-static" if Version(self.version) >= "2.20.0" and not self.options.shared and is_msvc(self) else ""
```